<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpDialog</class>
 <widget class="QDialog" name="SignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SignUpDialog {
	background-color: #7fcdee;
}

QLabel {
	color: #07407b;
}

#mainWidget QLabel {
	color: #ffffff;
	font-size: 14px;
	font-style: italic;
}

QLineEdit {
	border: 2.5px solid #f7931e;
	border-radius: 5px;
	padding: 7px 20px;
}

QComboBox {
	border: 2.5px solid #f7931e;
	border-radius: 5px;
	padding: 7px 20px;
}

QPushButton {
	color: #ffffff;
	font-weight: bold;
	background-color: #f7931e;
	border-radius: 5px;
	padding: 7px 20px;
}

QPushButton::hover {
	color:  #f7931e;
	font-weight: bold;
	background-color: #ffffff;
}

#newUserLabel {
	color: #07407b;
	font-weight: bold;
	font-size: 20px;
}

#mainWidget {
	background-color: #07407b;
	border: 2.5px solid #f7931e;
	border-radius: 15px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="newUserLabel">
     <property name="text">
      <string>New User</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="errorLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="usernameLabel">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="usernameLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="passwordLabel">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="confirmLabel">
          <property name="text">
           <string>Confirm Password:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="confirmLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="submit">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
